mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go

```diff
@@ -2128,6 +2128,235 @@ func TestAccSqlDatabaseInstance_ReplicaPromoteSkippedWithNoMasterInstanceNameAnd
 	})
 }
 
+func TestAccSqlDatabaseInstance_updateSslOptionsForPostgreSQL(t *testing.T) {
```

Comment:
Wanted to check: while these tests are nice and comprehensive, they are fairly expensive (primarily the parts where a new db instance is created). Is it important to check each of these combinations? If not, we might consider combining these tests, or merging them into an existing test (for example, just test `ssl_mode` being false vs true).

Reply:
SG. I only leave the one for PostgreSQL. No need for testing other DB types here because for TF they are the same.